### PR TITLE
Handle errors messages from delete response stream

### PR DESCRIPTION
**Pull request by romasku**
Adds support of https://github.com/neuromation/platform-storage-api/pull/239

### CODE REVIEW

File: neuromation/api/storage.py [hunk 1]
```diff
@@ -338,11 +338,18 @@ async def _rm(
         ) as resp:
             if resp.headers.get("Content-Type", "").startswith("application/x-ndjson"):
                 async for line in resp.content:
-                    remove_listing = json.loads(line)
+                    server_message = json.loads(line)
+                    if "error" in server_message:
+                        err_text = server_message["error"]
+                        os_errno = server_message.get("errno", None)
+                        if os_errno is not None:
+                            os_errno = errno.__dict__.get(os_errno, os_errno)
```
Comment (asvetlov) [inline neuromation/api/storage.py:346]: I'm missed. What is `errno` type here?

Comment (romasku) [reply neuromation/api/storage.py:346]: `errno` here is `module` instance (it comes from `import errno`). This is code is almost same as in `_Core.request`.

Comment (asvetlov) [reply neuromation/api/storage.py:346]: Got you

**Review by asvetlov — APPROVED** [review]

**Review by serhiy-storchaka — COMMENTED** [review]
Could you please add similar code for other response stream (`storage ls`, `job ls`)?

**Pull request merged**